crates/swc_ecma_transforms_module/src/umd.rs

```diff
@@ -508,4 +536,12 @@ impl Umd {
 
         (adapter_fn_expr, factory_params)
     }
+
+    fn pure_span(&self) -> Span {
+        let span = Span::dummy_with_cmt();
```

Comment:
I think this should go into the `if let` block below because `Span::dummy_with_cmt` is a heavy operation